src/evaluation/evaluation_environment.rs

```diff
@@ -659,28 +659,70 @@ impl EvaluationEnvironment {
         // to define inside of the expression
         let allowed = rhai_engine.eval_expression::<bool>(expression.as_str())?;
 
+        // The API Server puts some limitations on the warnings:
+        // - they cannot exceed 256 characters
+        // - the size of all the warnings cannot exceed 4096 characters
+        // - they are returned as HTTP headers, hence not all characters are allowed
+        //
+        // Because of these reasons, we use the warning struct only to
+        // tell the user whether a member policy was evaluated or not. When it was
+        // evaluated we just tell the outcome (allow/reject).
+        let mut warnings = vec![];
+
+        // The details of each policy evaluation are returned as part of the
+        // AdmissionResponse.status.details.causes
+        let mut status_causes = vec![];
+
+        let evaluation_results = policies_evaluation_results.lock().unwrap();
+
+        for policy_id in &policy_ids {
+            match evaluation_results.get(policy_id) {
```

Comment:
nit: we could use `if let` to reduce indentation

Reply:
done :+1: